ush/sample_configs/others/config.sh.RRFS_CONUS_3km

```diff
@@ -122,11 +127,11 @@ RUN="rrfs"
 
 . set_rrfs_config.sh
 
-STMP="YourOwnSpace/${version}/stmp"  # Path to directory STMP that mostly contains input files.
-PTMP="YourOwnSpace/${version}"  # Path to directory STMP that mostly contains input files.
-NWGES="YourOwnSpace/${version}/nwges"  # Path to directory NWGES that save boundary, cold initial, restart files
+STMP="${EXPT_BASEDIR}/stmp"  # Path to directory STMP that mostly contains input files.
```

Comment:
Some retros may not use ${EXPT_BASEDIR} as stmp/ptmp directory. Please do not change it.

Reply:
Hi @hu5970, changing these lines does not force anyone to use ${EXPT_BASEDIR} as stmp/ptmp directory, just saves a couple steps for everyone who does. Unless YourOwnSpace is a drive on a machine I'm unfamiliar with, having YourOwnSpace on these lines saves steps for no one.